examples/scenes/src/lib.rs

```diff
@@ -34,6 +40,8 @@ pub struct SceneParams<'a> {
     pub resolution: Option<Vec2>,
     pub base_color: Option<vello::peniko::Color>,
     pub complexity: usize,
+    #[cfg(feature = "cosmic_text")]
+    pub cosmic_text_scene_state: &'a CosmicTextSceneState,
```

Comment:
This is underdocumented, so I can understand this being unclear, but test scenes can have state. 

The main result of this is that that we can make this feature gate be "localised", by making the `cosmic_text_scene::text_scene` function return an `impl TestScene`, which will be retained.
For an example, see the `MMark` scene.

This does lead to some things which are unidiomatic, such as not sharing the fonts between the scenes, but I think for this peripheral functionality, keeping it as localised as possible is better than improving the efficiency.

Reply:
Done.